From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage due to enum addition for ath10k
Patch-mainline: Never, kABI fix
References: bsc#1051510

The patch patches.drivers/ath10k-update-the-phymode-along-with-bandwidth-chang
adds a new member to enum, and it's enough to get a complaint from kabi
checker.  Hide it from the nagging script.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/ath/ath10k/wmi.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/wireless/ath/ath10k/wmi.h
+++ b/drivers/net/wireless/ath/ath10k/wmi.h
@@ -5822,7 +5822,9 @@ enum wmi_peer_param {
 	WMI_PEER_NSS        = 0x5,
 	WMI_PEER_USE_4ADDR  = 0x6,
 	WMI_PEER_DEBUG      = 0xa,
+#ifndef __GENKSYMS__
 	WMI_PEER_PHYMODE    = 0xd,
+#endif
 	WMI_PEER_DUMMY_VAR  = 0xff, /* dummy parameter for STA PS workaround */
 };
 
